src/commands/entities/destroy.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as MixFlags from '../../utils/flags'
+import * as EntitiesAPI from '../../mix/api/entities'
+import MixCommand from '../../utils/base/mix-command'
+import {MixClient, MixResponse, EntitiesDeleteParams} from '../../mix/types'
+import {DomainOption} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:entities:destroy')
+
+export default class EntitiesDestroy extends MixCommand {
+  static description = `destroy an entity
+
+Use this command to permanently delete an entity from a project.`
+
+  static examples = [
+    '$ mix entities:destroy -P 1922 -E CoffeeSize',
+  ]
+
+  static flags = {
+    confirm: MixFlags.confirmFlag,
+    entity: MixFlags.entityFlag,
+    json: MixFlags.jsonFlag,
+    project: MixFlags.projectFlag,
+    yaml: MixFlags.yamlFlag,
+  }
+
+  action = 'destroy'
```

Comment:
As discussed, it's used to confirm the desctructive action with the user. So it's not set in non-desctructive commands but it could for consistency. Taking note of it.